R/read.any.R

```diff
@@ -43,6 +43,9 @@
 #' @param verbose Print helpful information via cat.
 #' @param widths Column widths. Only use for fixed width files.
 #' @param col.names Column names. Only use for fixed width files.
+#' @param sheets The word 'all', a vector of sheet names, or a vector of sheet numbers. Only use for excel files.
+#' @param pbiLocation pbi path
+#' @param pbiTable pbi table name
```

Comment:
Please use existing args here, too: `filename` instead of `pbiLocation` and `sheet` instead of `pbiTable`